packages/theming/win32-theme/src/createBrandedThemeWithAlias.ts

```diff
@@ -1,32 +1,34 @@
 import { Theme, PartialTheme, AliasColorTokens } from '@fluentui-react-native/theme-types';
 
-export function createBrandedThemeWithAlias(theme: Theme): PartialTheme {
-  if (!theme.host.colors) {
+export function createBrandedThemeWithAlias(themeName: string, theme: Theme): PartialTheme {
+  if (themeName === 'HighContrast' || !theme.host.colors) {
     return {};
```

Comment:
No values for HC yet?

Reply:
HC is in already (see https://github.com/microsoft/fluentui-react-native/tree/master/packages/theming/theme-tokens/src/highContrast) but we shouldn't apply brand colors in HC, so we skip this step.

Reply:
Specifically this steps pulls the brand color swatches directly from the native module, which doesn't change when you switch to HC.

Reply:
Ah I see, I never noticed that but thinking about it makes sense. Thanks for the explanation :)